src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 50;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+
+    private Random random = new Random();
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        switch (randomFigures()) {
+            case CIRCLE:
+                figure = new Circle(colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE));

```

Comment:
Create a private method for each type of figure. Like `getRandomCircle()`

Reply:
I don't see the need to do this, because the method is correct and easy to read